app/uk/gov/hmrc/tai/connectors/ApplicationUrls.scala

```diff
@@ -73,16 +73,21 @@ class BbsiUrls @Inject()(config: DesConfig) {
 }
 
 @Singleton
-class TaxAccountUrls @Inject()(npsConfig: NpsConfig, desConfig: DesConfig) {
+class TaxAccountUrls @Inject()(npsConfig: NpsConfig, desConfig: DesConfig, featureTogglesConfig: FeatureTogglesConfig) {
 
-  def taxAccountUrlNps(nino: Nino, taxYear: TaxYear): String =
-    s"${npsConfig.baseURL}/person/${nino.nino}/tax-account/${taxYear.year}/calculation"
+  private val desTaxAccountURL = (nino: Nino, taxYear: TaxYear) =>
+    s"${desConfig.baseURL}/pay-as-you-earn/individuals/${nino.nino}/tax-account/tax-year" +
+      s"/${taxYear.year}${if(!featureTogglesConfig.confirmedAPIEnabled)"?calculation=true" else ""}"
```

Comment:
Don't need the empty else here

Reply:
I think the if is evaluation to an expression